app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    our_list = []
+    for human in people:
+        person = Person(human["name"], human["age"])
```

Comment:
The `Person` class constructor is being called with `human["name"]` and `human["age"]`. Ensure that these keys are always present in the `human` dictionary to avoid potential `KeyError` exceptions. You might want to handle cases where these keys are missing.